packages/core/tsconfig.json

```diff
@@ -24,7 +24,6 @@
     "typeRoots": ["src/types", "node_modules/@types"],
     "esModuleInterop": true,
     "isolatedModules": true,
-    "noEmit": true,
```

Comment:
Does this affect the existing build scripts?

Reply:
good question, I did a quick check with rollup and I believe it overrides this flag anyways. But I am indeed not sure why it was there to begin with, perhaps an artifact of CRA.